frame/nft-fractionalization/src/lib.rs

```diff
@@ -185,10 +190,10 @@ pub mod pallet {
 			ensure!(nft_owner == who, Error::<T>::NoPermission);
 
 			let pallet_account = Self::get_pallet_account();
-
 			Self::do_lock_nft(nft_collection_id, nft_id)?;
-			Self::do_create_asset(asset_id, pallet_account)?;
+			Self::do_create_asset(asset_id, pallet_account.clone())?;
```

Comment:
should do_create_asset be taking a reference to pallet_account rather than ownership?

Reply:
then we would need to deref it inside because of `T::Assets::create()`